lang/cs/Org.Apache.REEF.Bridge/DriverLauncher.cpp

```diff
@@ -0,0 +1,353 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+#include "InteropUtil.h"
+using namespace std;;
+
+typedef jint(JNICALL *JNI_CreateJavaVM_FN)(JavaVM **pvm, void **penv, void *args);
+LPCTSTR JAVA_HOME = L"JAVA_HOME";
+LPCTSTR JVM_DLL1 = L"\\jre\\bin\\server\\jvm.dll";
+LPCTSTR JVM_DLL2 = L"\\jre\\bin\\client\\jvm.dll";
+const char* JNI_CreateJavaVM_Func_Name = "JNI_CreateJavaVM";
+const char* JavaOptionClassPath = "-classpath";
+const char* JavaMainMethodSignature = "([Ljava/lang/String;)V";
+const char* JavaStringClassPath = "java/lang/String";
+const char* NativeInteropClass = "org/apache/reef/javabridge/NativeInterop";
+const int maxPathBufSize = 16 * 1024;
+const char ClassPathSeparatorCharForWindows = ';';
+
+const char* launcherClass = "org.apache.reef.runtime.common.REEFLauncher";
+const char* JavaMainMethodName = "main";
+const char* JavaClassPath = "-Djava.class.path=";
+
+
+typedef enum {
```

Comment:
Will these every show up in error prints? If so, it would be good to have some documentation here.